Formula/sophus.rb

```diff
@@ -4,6 +4,7 @@ class Sophus < Formula
   url "https://github.com/strasdat/Sophus/archive/refs/tags/v22.10.tar.gz"
   sha256 "270709b83696da179447cf743357e36a8b9bc8eed5ff4b9d66d33fe691010bad"
   license "MIT"
+  revision 1
```

Comment:
We could probably do without the `revision` bump.

OTOH, there probably didn't need to be one when `fmt` was removed (and I wonder if we needed to remove it in the first place).

Reply:
The library is header-only, but the generated cmake config is different depending on the value of `SOPHUS_USE_BASIC_LOGGING`. See:
- https://github.com/strasdat/Sophus/blob/8e1941677ef214aab0397608f42b18fbe1a19885/CMakeLists.txt#L120
- https://github.com/strasdat/Sophus/blob/master/SophusConfig.cmake.in

To my understanding the revision bump ensures users get an update on the next `brew upgrade`.

Thanks for the workaround when things broke upstream!